### PR TITLE
Add a recommendation for running etcd at scale

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds info in the Scalability and Performance guide about
the maintenance operations needed for etcd to perform better.

### CODE REVIEW

**Comment by akrzos** [comment]
cc @hexfusion

File: modules/recommended-etcd-practices.adhoc
Comment (ahardin-rh) [inline modules/recommended-etcd-practices.adhoc]: Do you want this as part of the scalability_and_performance/recommended-host-practices.adoc file or as a new, standalone topic? If it's a new topic, then we need to add it to the _topic_map.yml file

Comment (ashishkamra) [reply modules/recommended-etcd-practices.adhoc]: In 3.11, etcd best practices were part of recommended-host-practices. So I think we can have the etcd guidelines at the same place in 4.1 as well.

Comment (chaitanyaenr) [reply modules/recommended-etcd-practices.adhoc]: Agreed, added this to the recommended-host-practices.adhoc file.

**Comment by ahardin-rh** [comment]
Once we get QE approval from @mffiedler this can be merged :+1:

**Review by mffiedler — APPROVED** [review]
/lgtm

File: modules/recommended-etcd-practices.adhoc
Comment (ahardin-rh) [inline modules/recommended-etcd-practices.adhoc]: @chaitanyaenr  I just noticed that the file needs to be renamed .adoc (not .adhoc).

File: scalability_and_performance/recommended-host-practices.adoc
Comment (ahardin-rh) [inline scalability_and_performance/recommended-host-practices.adoc]: .adoc not .adhoc

File: modules/recommended-etcd-practices.adhoc
Comment (chaitanyaenr) [reply modules/recommended-etcd-practices.adhoc]: Fixed it, thanks.

**Comment by ahardin-rh** [comment]
@chaitanyaenr Thanks! Merging!

**Comment by chaitanyaenr** [comment]
Thanks for the review.

**Pull request merged**
